addons/account_edi/models/account_edi_document.py

```diff
@@ -32,6 +34,17 @@ class AccountEdiDocument(models.Model):
         ),
     ]
 
+    def _compute_edi_content(self):
+        for document in self:
+            document.edi_content =  base64.b64encode(document.edi_format_id._get_edi_content(document.move_id))
```

Comment:
It should be called only if the document is `to_send` or `to_cancel`

Reply:
I've changed it but I wonder why? the download button is only visible if there is an error - it should be ok to compute this field using shell for example after it has been sent.